gui/src/components/GameTabs.svelte

```diff
@@ -0,0 +1,50 @@
+<script>
+  import { currentGame } from "../currentGame";
+
+  console.log("current game", $currentGame);
+
+  // TODO: Update version list when changing game
+  $: {
+  }
+</script>
+
+<div class="row">
+  <h1 class="heading">Games:</h1>
+  <button
+    type="button"
+    class="nes-btn is-warning btn"
+    class:active={$currentGame == "jumpy"}
```

Comment:
it would be nice if we use [strict equal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)(`===`) here and other places